### DISCUSSION TITLE
Re-Run Search

### DISCUSSION THREAD

**Discussion by utkonos** [Vector35/binaryninja-api#5264] [Ideas] [answered]
The new search interface shows the results of a search on a tab in the new search panel. For some types of search, I am looking for a pattern that I then want to change. Once I have made all the requisite changes, it would be helpful to re-run the exact same search from a single click. This way I can verify the changes that I made.

Would it make sense to have a little 🔄 button on the search tab to re-run that particular search without needing to CTRL-F and click Find again? This would be especially helpful if the search one wants to repeat is a prior search that is not the immediate prior. Under this condition, the user may even need to change components of the search in the interface before clicking Find. The re-run search button would cut out all those extra steps.

**Comment by plafosse** [comment, marked as answer]
Thanks for the question this is a good feature request. Created an issue: https://github.com/Vector35/binaryninja-api/issues/5286
